File: program/convert-seconds-to-minutes/convertSecondsTominutes.js
let seconds = 60;
let minutes = Math.floor(seconds / 60);
let extraSeconds = seconds % 60;
minutes = minutes < 10 ? "0" + minutes : minutes;
extraSeconds = extraSeconds < 10 ? "0" + extraSeconds : extraSeconds;
console.log(`${minutes}mins : ${extraSeconds}secs`);

File: program/convert-seconds-to-minutes/convertSecondsTominutes.py
import math
seconds = 1000;
minutes = math.floor(seconds / 60);
extraSeconds = seconds % 60;
if minutes < 10:
    minutes =  "0" + str(minutes)
        
if extraSeconds < 10:
    extraSeconds =  "0" + str(extraSeconds) 
        
print(f"{minutes}mins : {extraSeconds}secs");